src/com/jalasoft/search/controller/SearchCriteria.java

```diff
@@ -0,0 +1,321 @@
+/*
+ *  Copyright (c) 2018 Jalasoft.
+ *
+ *  This software is the confidential and proprietary information of Jalasoft.
+ *  ("Confidential Information").  You shall not disclose such Confidential Information and shall use
+ *   it only in accordance with the terms of the license agreement you entered into with Jalasoft.
+ */
+
+package com.jalasoft.search.controller;
+
+/**
+ * SearchCriteria : This class is a container class when will be stored the search criteria
+ *
+ * @author rafael alfaro
+ * @version 1.0
+ */
+
+public class SearchCriteria {
+
+    private String driver;
+    private String searchPath;
+    private String fileName;
+    private String advanceSearch;
+    private String contains;
+    private boolean inTitle;
+    private boolean insideFile;
+    private String ownerFile;
+    private boolean fileCreated;
+    private boolean fileModified;
+    private boolean fileAccessed;
```

Comment:
group variables by type and take into account the java conventions: i.e.
String   driver, searchPath, fileName,
             advancedSearch, contains;

Reply:
Updated to:
    // Attribute used on basic Search
    private String driver;
    private String searchPath;
    private String fileName;
    // Attribute used on advanced search
    private String advanceSearch;
    private String contains;
    private String ownerFile;
    private String sizeFile;
    private String sizeType;
    private char sizeCriteria;
    private boolean inTitle;
    private boolean insideFile;
    private boolean fileCreated;
    private boolean fileModified;
    private boolean fileAccessed;
    private boolean fileHidden;
    private boolean directory;